src/frontend/src/app/issue/issue.component.html

```diff
@@ -1,54 +1,57 @@
 <div class="issue-creation">
-    <a class="text-create" (click)="issueModal.show()" id="btn-create-issue">Create issue</a>
- 
-    <div bsModal #issueModal="bs-modal" class="modal fade" tabindex="-1" role="dialog" aria-labelledby="issueModal" aria-hidden="true">
-        <div class="modal-dialog modal-lg">
-            <div class="modal-content">
-                <div class="modal-header">
-                    <h4 class="modal-title pull-left">Create issue</h4>
-                    <button type="button" class="close pull-right" (click)="issueModal.hide()" aria-label="Close">
-                    <span aria-hidden="true">&times;</span>
-                    </button>
-                </div>
-                <div class="modal-body">
-                    <form>
-                        <label for="summary">Summary<span class="required-star">*</span></label><br>
-                        <input name="summary" [(ngModel)]="issueDetails.summary"><br>
-                            
-                        <label for="description">Description</label><br>
-                        <textarea name="description" cols="50" [(ngModel)]="issueDetails.description"></textarea><br>
-                            
-                        <label for="assignees">Assignees</label><br>
-                        <div class="assignees-multi-select" *ngIf="projectMembers">
-                            <angular2-multiselect [data]="projectMembers" 
-                            [(ngModel)]="selectedItems" 
-                            [settings]="settings"
-                            name="assignees">
-                            </angular2-multiselect>
-                        </div><br>
+  <a class="text-create" (click)="issueModal.show()" id="btn-create-issue">Create issue</a>
```

Comment:
to github tak pokazuje? bo na dole mam pusto i dużo odstępu, dziwnie

Reply:
mam chyba inne formatowanie odstępów i takie rzeczy wychodzą
![image](https://user-images.githubusercontent.com/25505958/37951760-d297535c-319d-11e8-8b56-740e9cdcccf2.png)